.github/workflows/cd.yml

```diff
@@ -79,4 +82,18 @@ jobs:
               repo: context.repo.repo,
               release_id: '${{ needs.build.outputs.release_id }}',
               name: '${{ github.ref_name }}',
-            })
\ No newline at end of file
+            })
+
+      - name: Publish distribution 📦 to Test PyPI
+        if: github.repository == 'vmware/repository-service-tuf-cli'
+        uses: pypa/gh-action-pypi-publish@0bf742be3ebe032c25dd15117957dc15d0cfc38d
+        with:
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+          repository-url: https://test.pypi.org/legacy/
+
+      - name: Publish binary wheel and source tarball 📦 on PyPI
+        # Only attempt PyPI upload in upstream repository
+        if: github.repository == 'vmware/repository-service-tuf-cli'
+        uses: pypa/gh-action-pypi-publish@0bf742be3ebe032c25dd15117957dc15d0cfc38d
+        with:
+          password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
```suggestion
```
We can use [trusted publishing](https://github.com/marketplace/actions/pypi-publish#trusted-publishing) with this action.
I already configured it on PyPI.